tests/platforms/windows/app/test_package.py

```diff
@@ -27,12 +28,50 @@ def package_command(tmp_path):
         version="81.2.1.0",
         arch="groovy",
     )
+    command.tmp_path = tmp_path
     return command
 
 
+@pytest.fixture
+def package_command_with_files(package_command):
+    # Build the path for the source folder:
+    src_path = (
+        package_command.tmp_path
+        / "base_path"
+        / "build"
+        / "first-app"
+        / "windows"
+        / "app"
+        / "src"
+    )
+    src_path.mkdir(parents=True)
+
+    # Mock some typical folders and files in the src folder:
+    paths = (
+        "app/first-app/resources/",
+        "app/first-app-0.0.1.dist-info/",
+        "app_packages/toga_winforms/",
+    )
+    package_command.files = (
+        "First App.exe",
+        "python.exe",
+        "python3.dll",
+        "vcruntime140.dll",
+        "app/first-app/app.py",
+        "app/first-app/resources/__init__.py",
+        "app_packages/clr.py",
+        "app_packages/toga_winforms/command.py",
```

Comment:
Although this evidently works on Windows, it would be preferable to use Pathlib objects and make the path separator unambiguous.

Reply:
I'd prefer to let it stand as it is here; this list is also used to check the content of the ZIP file, and the namelist of the ZIP file consists of strings in this format.
However, the entries will be converted to Pathlib objects when creating the file with `create_file()`, as you suggested.

Reply:
But that's not what this part of the test is for. It's creating fixtures. It's *essential* that they are created with the correct, platform appropriate path separators. The test fixture should *not* also be providing the test validation data - the test should be explicitly specifying what it expects, not relying on the input from a fixture.

Reply:
This is especially important when what is being validated isn't in the same format as what is being put into the system in the first place.

Reply:
OK, changed it.